evm/src/trace/identifier/etherscan.rs

```diff
@@ -57,13 +106,17 @@ impl TraceIdentifier for EtherscanIdentifier {
             for (addr, _) in addresses {
                 fetcher.push(*addr);
             }
```

Comment:
We can probably filter out addresses here if it is already present in `self.contracts`